main/spl/APKBUILD

```diff
@@ -17,6 +17,13 @@ source="https://github.com/zfsonlinux/zfs/releases/download/zfs-$pkgver/spl-$pkg
 
 builddir="$srcdir"/spl-$pkgver
 
+prepare() {
+	default_prepare || return 1
+	# Current package version does not contain information about ppc64le on
+	# config.sub and config.guess. Calling autoreconf to update these files.
```

Comment:
Aha, then it’s not needed to call `autoreconf`, we have two helper functions for this: `update_config_sub && update_config_guess`.